configure

```diff
@@ -6742,9 +6742,7 @@ fi
   *) :
 
 			# other host types: look for DWARF support
-			if test "x$OMR_CROSS_CONFIGURE" != "xyes"
-			then
-				for ac_header in dwarf.h libdwarf/dwarf.h libdwarf.h libdwarf/libdwarf.h
+			for ac_header in dwarf.h libdwarf/dwarf.h libdwarf.h libdwarf/libdwarf.h
```

Comment:
I believe this file is derived from `configure.ac` which should be similarly updated so this is not lost in a future change.

Reply:
I didn't notice the file `configure.ac`. Thank you for pointing it out.